config/routes.rb

```diff
@@ -44,6 +44,12 @@
     member do
       put :delete_reason
     end
+
+    collection do
+      get "multi/step/:step", to: "multi_transfers#step"
+      post "multi/step/:step", to: "multi_transfers#step"
```

Comment:
these 2 routes can be unified by:

`match "multi/step/:step", to: "multi_transfers#step", via: [:get, :post]`